frontend/src/pages/add-item/index.js

```
@@ -1,87 +1,74 @@
+
+import { createBrowserHistory } from "history";
 import React, { useState } from "react";
-import FormInput from "../../components/InputText";
-import Button from "../../components/Button";
-import InputDate from "../../components/InputDate";
-import DropDown from "../../components/DropDownMenu";
-import NavBar from "../../components/NavBar";
+import NavBar from "../../components/navbar";
+import { Modal } from "semantic-ui-react";
 import "./index.css";
+import axios from "axios";
+
 
-const usersOptions = [
-  {
-    id: 1,
-    user: "user-1",
-  },
-  {
-    id: 2,
-    user: "user-2",
-  },
-];
 
 const AddItemPage = () => {
-  const [itemName, setItemName] = useState("");
-  const [room, setRoom] = useState("");
-  const [reserve, setReserve] = useState("");
-  const [fromDateInput, setFromDateInput] = useState("");
-  const [toDateInput, setToDateInput] = useState("");
+  const [name,setItem] = useState("")
+  const [room,setRoom] = useState("")
+  const history = createBrowserHistory()
+ 
+  const addItem = async () =>{
+    let formField = new FormData();
+    formField.append("name",name)
+    formField.append("room",room)
 
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    console.log(`
-    Item Name: ${itemName}
-    Room: ${room}
-    Reserved User: ${reserve}
-    From: ${fromDateInput}
-    To: ${toDateInput}
-    `);
-  };
+   axios({
+    method: 'post',
+    url:"http://localhost:8000/api/items/",
+    data: formField}).then((response)=>console.log(response.data));
```

maybe add success/fail message after requests after sending data to BE instead of just console log